### PR TITLE
move test-runner modules part 2

### PR DESCRIPTION
**Pull request by dsvoronin**
Second part of #982

AFAIK this is all test runner modules

### CODE REVIEW

File: subprojects/settings.gradle.kts [hunk 1]
```diff
@@ -99,31 +92,36 @@ include(":common:truth-extensions")
...
 
 include(":android-lib:proxy-toast")
 include(":android-lib:snackbar-proxy")
 
+include(":test-runner:test-report-artifacts")
+include(":test-runner:test-annotations")
+include(":test-runner:test-report-api")
+include(":test-runner:test-report-dsl-api")
+include(":test-runner:test-report-dsl")
+include(":test-runner:test-report")
```
Comment (eugene-krivobokov) [inline subprojects/settings.gradle.kts:117]: How about grouping `test-report` modules?

```
test-report/
  api
  dsl-api
  dsl
  report
```

Comment (dsvoronin) [reply subprojects/settings.gradle.kts:117]: not sure for now; let's draw a whole target picture before any changes (besides moving to test-runner xd)

Comment (eugene-krivobokov) [reply subprojects/settings.gradle.kts:117]: > let's draw a whole target picture before any changes

Agree. Is it applicable to this PR also?

**Review by eugene-krivobokov — APPROVED** [review]

File: subprojects/settings.gradle.kts (hunk 1 above)
Comment (dsvoronin) [reply subprojects/settings.gradle.kts:117]: not this PR

**Pull request merged**